src/javascript/app/base/client.js

```diff
@@ -116,10 +116,10 @@ const Client = (() => {
     });
 
     const doLogout = (response) => {
-
+     
```

Comment:
empty tab?

Reply:
Removed. I used eslint to format code, but strangely it left spaces.